test/unit/vdom/skip.js

```diff
@@ -0,0 +1,60 @@
+import { define, prop, props, symbols, vdom } from '../../../src/index';
+import afterMutations from '../../lib/after-mutations';
+import fixture from '../../lib/fixture';
+
+describe('vdom/skip', () => {
+  it('should skip the element children', done => {
+    const Elem = define('x-test', {
+      props: {
+        num: prop.number()
+      },
+      render() {
+        vdom.element('div', { skip: true }, () => {
+          vdom.element('span', () => {
+            vdom.text('text');
+          });
+        });
+      },
+    });
+    const elem = new Elem();
+    fixture(elem);
+    afterMutations(
+      () => expect(elem[symbols.shadowRoot].textContent).to.equal(''),
+      () => props(elem, { num: elem.num + 1 }),
+      () => expect(elem[symbols.shadowRoot].textContent).to.equal(''),
+      done
+    );
+  });
+
+  it('should allow conditional rendering', done => {
+    function isEven(num) {
+      return num % 2 === 0;
+    }
+    const Elem = define('x-test', {
+      props: {
+        num: prop.number()
```

Comment:
I assume the default value is 0. It might be nice to add that explicitly `{ default: 0 }` to remove any ambiguity.